import 'package:task_save/core/errors/failure.dart';

class TaskNotFoundFailure extends Failure {
  const TaskNotFoundFailure();
}
